geo/geo.go

```diff
@@ -40,8 +41,12 @@ func (l *Location) Match(location *Location) bool {
 }
 
 type IPFilter struct {
-	Allow    []Location `json:"allow"`
-	Disallow []Location `json:"disallow"`
+	Allow       []Location    `json:"allow"`
+	Disallow    []Location    `json:"disallow"`
+	AWSFile     string        `json:"awsfile"`
```

Comment:
It's better to let user configure a directory and put all geo files there

Reply:
User can config geo files in config files in `IPFilter` scope